packages/mongo/collection/collection.js

```diff
@@ -165,6 +164,87 @@ Object.assign(Mongo.Collection.prototype, {
     return this._connection && this._connection !== Meteor.server;
   },
 
+  /**
+   * @summary Watch MongoDB Change Streams and publish changes via DDP.
+   * @locus Server
+   * @memberof Mongo.Collection
+   * @instance
+   * @param {Object} subscription The DDP subscription object
+   * @param {Array} [pipeline] Optional aggregation pipeline to filter Change Stream events.
+   * @param {Object} [options] Optional settings for the Change Stream.
+   * @returns {Object} Handle with stop() method to stop watching.
+   * @throws {Error} If called on a client/minimongo collection.
+   *
+   * @example
+   *   // In a publish function:
+   *   Meteor.publish('myChanges', function() {
+   *     return MyCollection.watch(this, [
+   *       { $match: { 'operationType': 'insert' } }
+   *     ]);
+   *   });
+   */
+  watch(subscription, pipeline = [], options = {}) {
```

Comment:
How would this snippet below work in this `watch` example?
```js
Meteor.publish('myChanges', function() {
       return MyCollection.find();
})
```

Also, passing the `this`, seems a bit weird; maybe the API could be `MyCollection.watch(pipeline, options)`. The `Publish` method could check if it is a `watch` method, and if it is, run the necessary operations

Also2, would it work like we do for [array of cursors?](https://arc.net/l/quote/ajibbhyo)

Reply:
for this I'm awaitting a new PR from @rodrigooler 
So disregard it for a while